source/_components/media_player.yamaha.markdown

```diff
@@ -29,14 +29,18 @@ To add a Yamaha Network Receiver to your installation, add the following to your
 # Example configuration.yaml entry
 media_player:
   platform: yamaha
+  host: 192.168.1.100
   name: 'Basement Receiver'
 ```
 Configuration variables:
 
 - **name** (*Optional*): Name of the device
+- **host** (*Optional*): IP address of the device
```

Comment:
Can this be also be a hostname or only an IP address? If so the extended text below should probably be updated to reflect this.